pennylane/vqe/vqe.py

```diff
@@ -542,6 +551,12 @@ def metric_tensor(self, args, kwargs=None, diag_approx=False, only_construct=Fal
         Returns:
             array[float]: metric tensor
         """
+        if self._multiple_devices:
+            warnings.warn(
+                "ExpvalCost was instantiated with multiple devices. The first device "
```

Comment:
```suggestion
                "ExpvalCost was instantiated with multiple devices. Only the first device "
```
Not sure if it reads better like this or not? Feel free to reject.